lutris/runners/mame.py

```diff
@@ -41,6 +54,14 @@ def config_dir(self):
     def working_dir(self):
         return self.config_dir
 
+    def get_platform(self):
+        selected_platform = self.game_config.get("platform")
+        if selected_platform:
+            return self.platforms[int(selected_platform)]
+        else:
+            return ("Arcade")
+        return ""
```

Comment:
This return will never be called. There is already a return for both if and else.

Reply:
I found that if I don't include this, current games will loose their platform, if nothing gets manually changed. Also if you do not choose a platform at the time you create the entry, you will end up without a platform, if I remove the "else:". With it in the platform gets auto-assigned to arcade. I tested a few times this morning, trying to get it to auto-convert games already in lutris

Reply:
Think I fixed it correctly :D